security/pkg/workload/server.go

```diff
@@ -90,11 +90,16 @@ func (s *SDSServer) SetServiceIdentityPrivateKey(content []byte) error {
 	return nil
 }
 
-// Save saves the specified key cert.
-func (s *SDSServer) Save(b util.KeyCertBundle) error {
+// Put stores the KeyCertBundle for a specific service account.
+func (s *SDSServer) Put(serviceAccount string, b util.KeyCertBundle) error {
 	return nil
 }
 
+// Get returns the KeyCertBundle for a specific service account.
+func (s *SDSServer) Get(serviceAccount string) (util.KeyCertBundle, error) {
```

Comment:
and we don't need this function for now as well, right?

Reply:
yes. not needed for now.